daprdocs/content/en/developing-applications/building-blocks/state-management/state-management-overview.md

```diff
@@ -3,7 +3,7 @@ type: docs
 title: "State management overview"
 linkTitle: "Overview"
 weight: 100
-description: "Overview of the state management building block"
+description: "Overview of the state management building block API"
```

Comment:
to confirm with the other naming conventions (e.g. see `building-blocks/service-invocation/service-invocation-overview.md `)
```suggestion
description: "Overview of the state management API building block"
```